pkg/bicep/bicep.go

```diff
@@ -23,19 +29,109 @@ var template = map[string]interface{}{
 }
 
 func GenerateFiles(buildPath, stepPath string, b *bundle.Bundle, fs afero.Fs) error {
-	p := path.Join(buildPath, stepPath, paramsFile)
+	p := path.Join(buildPath, stepPath, ParamsFile)
 	params, _ := inputvars.TranspileDevParams(p, b, fs)
 	connections, _ := inputvars.TranspileConnectionVarFile(p, b, fs)
 
-	template["parameters"].(map[string]interface{})["params"] = params
-	template["parameters"].(map[string]interface{})["md_metadata"] = inputvars.DefaultMdMetadata(b.Name)
-	template["parameters"].(map[string]interface{})["connections"] = connections
+	template["parameters"].(map[string]interface{})["params"] = map[string]any{"value": params}
```

Comment:
Should this logic eventually be moved to airlock?

Reply:
I am not totally sure what airlock does. so maybe?

Reply:
Translates to/from JSON Schema and IaC DSL.

JSON Schema <-> Terraform variables
JSON Schema <-> Helm values.yaml

Reply:
Bicep just takes a raw json blob with a values key. I guys that fits.

Reply:
We can move it to airlock later. Just wanted to clarify.